### PR TITLE
Update the OCI experimental tool to match the current Tekton bundle f…

### PR DESCRIPTION
**Pull request by pierretasci**
# Changes

Updates the OCI tool to use the new Tekton Bundle format. This allows users to test out the functionality with this tool before an official Tekton Bundle tool is built.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: oci/go.mod [hunk 1]
```diff
@@ -13,14 +13,12 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.4.0 // indirect
 	github.com/spf13/cobra v0.0.5
-	github.com/tektoncd/pipeline v0.10.1
+	github.com/tektoncd/pipeline v0.11.2
```
Comment (vdemeester) [inline oci/go.mod:16]: Any reason for it to be `v0.11.2` instead of a later version ?

**Review by vdemeester — APPROVED** [review]

File: oci/go.mod (hunk 1 above)
Comment (pierretasci) [reply oci/go.mod:16]: That's the version when I first wrote this modification to the experimental tool  👀

Comment (vdemeester) [reply oci/go.mod:16]: ok, let's get this in anyway, we can bump this in a follow-up :wink:

**Pull request merged**